docs/installation.rst

```diff
@@ -48,6 +48,9 @@ You must have root privilege :
 
         sudo apt-get update
         sudo apt-get install netcdf-bin libnetcdf-c++4-dev libboost-all-dev cmake
+        svn checkout http://forge.ipsl.jussieu.fr/ioserver/svn/XIOS/trunk xios
+        cd xios
+        ./make_xios --arch <your_architecture>
```

Comment:
thanks but this is no longer required (`--netcdf_lib netcdf4_seq`) because I have changed the way xios is built. I believe this was a workaround anyway. So now we do not need it.